.github/workflows/ci.yml

```diff
@@ -0,0 +1,27 @@
+on: [push, pull_request]
+name: Test
+jobs:
+  test:
+    strategy:
+      fail-fast: false
+      matrix:
+        rvm:
+          - 2.6
+          - 2.5
+          - 2.4
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ruby/setup-ruby@v1
+        env:
+          BUNDLE_GEMFILE: ${{ github.workspace }}/Gemfile
+        with:
+          ruby-version: ${{ matrix.rvm }}
+          bundler-cache: true
+          # Run bundle install outside the workspace to keep gems out of the
+          # repo checkout.
+          working-directory: ${{ runner.temp }}
+      - run: |
+          bundle exec rake
+      # TODO: notifications
```

Comment:
```suggestion
      # TODO: send email notifications
```

Reply:
I've actually embedded the whole Travis notification section in the TODO instead — that way we know exactly what needs to be replicated…

Reply:
Sounds great